Sprint-1/exercises/paths.js

```diff
@@ -15,4 +15,8 @@ const base = filePath.slice(lastSlashIndex + 1);
 console.log(`The base part of ${filePath} is ${base}`);
 
 // Create a variable to store the dir part of the filePath variable
+let dirPart = filePath.slice(7,44)
+console.log(dirPart)
 // Create a variable to store the ext part of the variable
+let extPart = filePath.slice(-4)
+console.log(extPart)
```

Comment:
Can you modify these code so that they can work for different file paths? For examples,

`const filePath = "/path1/path2/package.json";` or 
`const filePath = "/path1.something/.git/something.md";`